application/features/bootstrap/EmailContext.php

```diff
@@ -669,6 +736,22 @@ public function iSeeThatAMfaDisabledEmailShouldBeSent()
         assert::true($this->mfaDisabledEmailShouldBeSent);
     }
 
+    /**
+     * @Then I see that a get backup codes email has NOT been sent recently
+     */
+    public function iSeeThatAGetBackupCodesEmailHasNotBeSent()
+    {
+        assert::false($this->getBackupCodesEmailHasBeenSent);
```

Comment:
Since this would be a simple fix I'll mention it, even though the code apparently works either way:
All of these `assert::` calls should probably be `Assert::` since class names in PHP use TitleCase.

Reply:
Done